---
title: 转录组原始数据处理流程
author: Songqi Duan
date: '2020-04-19'
slug: raw_data_processing_flow
categories:
  - 生物信息学
tags:
  - Linux
lastmod: '2020-04-18T15:04:27+08:00'
keywords: []
description: ''
comment: yes
postMetaInFooter: yes
hiddenFromHomePage: no
contentCopyright: <a href='https://creativecommons.org/licenses/by-nc-nd/4.0/'>CC
  BY-NC-ND 4.0</a>
reward: no
mathjax: yes
mathjaxEnableSingleDollar: no
mathjaxEnableAutoNumber: no
hideHeaderAndFooter: no
flowchartDiagrams:
  enable: no
  options: ''
sequenceDiagrams:
  enable: no
  options: ''
output:
  blogdown::html_page:
    toc: yes
---


<div id="TOC">
<ul>
<li><a href="#mm10-gtf-gencode">mm10 gtf gencode</a></li>
</ul>
</div>

<p><a href="https://www.jianshu.com/p/a84cd44bac67" class="uri">https://www.jianshu.com/p/a84cd44bac67</a>
# 进行质控
nohup fastqc -t 4 ./<em>.gz -o ./fastqc_results &gt; ./fastqc.log 2&gt;&amp;1 &amp;
# 去除低质量的部分
nohup trim_galore –illumina -o clean raw/fastq/</em>.gz &amp;
nohup trim_galore -q 25 –phred33 –fastqc –length 36 –stringency 3 -o clean raw/fastq/<em>.gz &amp;
平台：–nextera
# 序列比对
index=/home/duansq/reference/index/hisat/mm10
ls ./</em>.gz | while read id;do
hisat2 -p 5 -x $index -U $id -S $(basename $id).bam.hisat.sam
done
# sam2bam
ls <em>.sam|while read id;do (samtools sort -O bam -@ 5 -o $(basename ${id} “.sam”).bam ${id});done
# gtf
gtf=/home/duansq/reference/gtf/gencode/mm10/gencode.vM24.annotation.gtf.gz
featureCounts -T 5 -p -t exon -g gene_id -a $gtf -o all.id.txt </em>.bam 1&gt;counts.id.log 2&gt;&amp;1 &amp;</p>
<div id="mm10-gtf-gencode" class="section level1">
<h1>mm10 gtf gencode</h1>
<p><a href="ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.annotation.gtf.gz" class="uri">ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.annotation.gtf.gz</a>
<a href="ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.chr_patch_hapl_scaff.annotation.gtf.gz" class="uri">ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.chr_patch_hapl_scaff.annotation.gtf.gz</a>
<a href="ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.primary_assembly.annotation.gtf.gz" class="uri">ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.primary_assembly.annotation.gtf.gz</a>
<a href="ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.basic.annotation.gtf.gz" class="uri">ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.basic.annotation.gtf.gz</a>
<a href="ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.chr_patch_hapl_scaff.basic.annotation.gtf.gz" class="uri">ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.chr_patch_hapl_scaff.basic.annotation.gtf.gz</a>
<a href="ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.long_noncoding_RNAs.gtf.gz" class="uri">ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.long_noncoding_RNAs.gtf.gz</a>
<a href="ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.polyAs.gtf.gz" class="uri">ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.polyAs.gtf.gz</a>
<a href="ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.2wayconspseudos.gtf.gz" class="uri">ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.2wayconspseudos.gtf.gz</a>
<a href="ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.tRNAs.gtf.gz" class="uri">ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M24/gencode.vM24.tRNAs.gtf.gz</a></p>
</div>
